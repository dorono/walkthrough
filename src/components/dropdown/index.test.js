import React from 'react';
import {shallow} from 'enzyme';
import Dropdown from './index';

jest.mock('react-onclickoutside');

const defaultProps = {
    onOptionClick: () => null,
    options: [],
    selected: {},
};

describe('Dropdown', () => {
    it('should render without errors', () => {
        shallow(<Dropdown {...defaultProps} />);
    });

    it('should match snapshot', () => {
        expect(shallow(<Dropdown {...defaultProps} />)).toMatchSnapshot();
    });
});
