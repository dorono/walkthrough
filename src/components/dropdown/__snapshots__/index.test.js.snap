// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dropdown should match snapshot 1`] = `
<div
  className="root"
>
  <div
    className="header"
    onClick={[Function]}
  >
    <span />
    <img
      alt="Open"
      className="arrow"
      src="test-file-stub"
    />
  </div>
  <OptionsMenu
    isOpen={false}
    onOptionClick={[Function]}
    options={Array []}
    portalNode={
      <div
        id="dropdown-portal-container"
      />
    }
    selected={Object {}}
    selectedClassName="selected"
  />
</div>
`;
